src/_includes/components/projects-grid.html

```diff
@@ -0,0 +1,16 @@
+<section class="projects-grid">
+  <div class="projects-grid__tag-nav">
+    {% for project in site.projects %}
```

Comment:
So this is what populates the tag selector. I have an idea for generating the tags: take `site.projects` and `reduce` into an array of unique tags. So if we have 30 projects with various tags, we can reduce over those 30 projects and get the array of tags.

Reply:
With way the page is designed to work (or intended) we don't need to worry about the tags. We tag each product object with the tags in a data attribute.